flask_common/mongo/utils.py

```diff
@@ -78,6 +83,10 @@ def fetch_related(
     model uses organization_id as a shard key, and all contacts are expected to
     be in the same organization, you can pass:
     {Contact: {'organization_id': organization.pk}}
+
+    The function takes an optional dict filter_funcs in the form
+    {document_class: filter_func} which represents the function that is used
```

Comment:
What do you think about passing in a simpler `document_class: queryset_attribute` dict, e.g. `{ConnectedAccount: 'all_objects'}` and then using `getattr` below?

Reply:
It sounds a bit less flexible and uses more "magic" since we're now passing strings that represent functions, why do you think this is better?

Reply:
I guess I was using a mental shortcut that doesn't really apply well here, namely that it's better to pass simple types than complex ones, but you're right that the "more complex" `ConnectedAccount.objects` is actually more accurate. Feel free to leave as-is.